build/dependencies.props

```diff
@@ -17,7 +17,7 @@
     <MicrosoftNETTestSdkPackageVersion>15.3.0</MicrosoftNETTestSdkPackageVersion>
     <MicrosoftVisualStudioComponentModelHostPackageVersion>15.0.26606</MicrosoftVisualStudioComponentModelHostPackageVersion>
     <MicrosoftVisualStudioEditorPackageVersion>15.0.26606</MicrosoftVisualStudioEditorPackageVersion>
-    <MicrosoftVisualStudioLanguageIntellisensePackageVersion>15.0.26606</MicrosoftVisualStudioLanguageIntellisensePackageVersion>
+    <MicrosoftVisualStudioLanguageIntellisensePackageVersion>15.6.161-preview</MicrosoftVisualStudioLanguageIntellisensePackageVersion>
```

Comment:
If we're updating one of these VS packages we should update all of them.

@natemcmaster - what's the value of listing all of these external dependencies with their own package versions? Can we just have a single variable for all of the things that version together (Roslyn, VS editor)

Reply:
Condensing variables into logical groups of packages that version together might make the dependencies.props file more user-friendly, but it makes it less bot-friendly. And at the moment, this would break our auto-update automation and the Universe build. We designed this file for simplicity: variables should be `packageId.ToPascalCase() + "PackageVersion"` and there should be one version variable per unique package. See https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/auto-dependency-flow/api.md. 

@NTaylorMullen make sure to update this: https://github.com/aspnet/Universe/blob/dev/build/dependencies.props#L171 when you merge. Otherwise the auto-update bot will rollback to a previous version. (cc @ryanbrandenburg we should probably make sure the Update-Repos step ensures repos compile before pushing any changes.)

Reply:
I filed aspnet/Universe#636 to track making our update script check that everything compiles.